src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +49,15 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
```

Comment:
Check if the movie session retrieved by `movieSessionService.get(yesterdayMovieSession.getId())` is not null before printing it to prevent `NullPointerException`.